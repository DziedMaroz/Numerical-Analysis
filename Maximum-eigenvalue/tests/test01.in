3
1   2   3
3   2   3
3   5   7
1   1   1
0.0000001
