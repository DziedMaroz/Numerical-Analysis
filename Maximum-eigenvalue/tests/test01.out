10.673392446138907
